$OpenBSD: patch-python_core_qgscoordinatetransform_sip,v 1.1 2017/07/19 15:48:31 zhuk Exp $
Fix build with py-sip-4.19.3.

Idea taken from Arch Linux, see https://aur.archlinux.org/packages/qgis/

Index: python/core/qgscoordinatetransform.sip
--- python/core/qgscoordinatetransform.sip.orig
+++ python/core/qgscoordinatetransform.sip
@@ -15,7 +15,6 @@
 class QgsCoordinateTransform : QObject
 {
 %TypeHeaderCode
-extern PyObject *sipExportedExceptions__core[2];  // workaround: sipExportedExceptions__core is only defined in the first sip part
 #include <qgscoordinatetransform.h>
 %End
 
